packages/eslint-config-ibmdotcom/package.json

```diff
@@ -1,7 +1,8 @@
 {
-  "name": "@ibmdotcom/eslint-config-ibmdotcom",
+  "name": "@carbon/eslint-config-ibmdotcom",
```

Comment:
Just thinking about naming strategy and consistency, all the others start with `ibmdotcom`. Should these be named like... `@carbon/ibmdotcom-eslint-config`?

Reply:
One other thought on the naming of the directory this lives in. As an example we don't include the name of the package in the folder name for `react`, but for namespace and context we include it in the package name on `npm`. I think that approach works, and cuts down redundancy since we already know we are within the `ibmdotcom` context when working locally. Should we do something similar for our `config` type packages here as well?

```
|
|-react
|-eslint-config
|-styles
|-stylelint-config
```

```
@carbon/ibmdotcom-react
@carbon/ibmdotcom-eslint-config
@carbon/ibmdotcom-styles
@carbon/ibmdotcom-stylelint-config
```

Reply:
@photodow good question. I was actually going that route, in fact was going with `config-eslint` and `config-stylelint` so in the package folder they would be essentially next to each other. But then I decided to change it back and have `ibmdotcom` be at the end, since I was looking at the other packages and they seem to have that pattern and probably was why carbon went with that naming pattern as well. e.g. `eslint-config-[name of package]` and `stylelint-config-[name of package]`. Looking at public ones like `eslint-config-airbnb` or `stylelint-config-prettier`, etc. ESlint even has documentation on `shareable configs` that follow this naming pattern, if we ever decide to share ours:

https://eslint.org/docs/developer-guide/shareable-configs